includes/updates/update-1.6.9.php

```diff
@@ -0,0 +1,73 @@
+<?php
+namespace WeDevs\ERP\HRM\Update;
+/*
+ * Add transaction_charge column in `erp_acct_pay_bill` table
+ */
+function erp_acct_alter_purchase_1_6_9() {
+    global $wpdb;
+
+    $table = $wpdb->prefix . 'erp_acct_purchase';
+    $cols  = $wpdb->get_col( "DESC $table" );
+
+    if ( !in_array( 'tax', $cols ) ) {
+        $wpdb->query(
+            $wpdb->prepare(
+                "ALTER TABLE $table ADD `tax` decimal(20,2)  NULL DEFAULT NULL  AFTER `amount`;"
+            )
+        );
+    }
+
+    if ( !in_array( 'tax_zone_id', $cols ) ) {
+        $wpdb->query(
+            $wpdb->prepare(
+                "ALTER TABLE $table ADD `tax_zone_id` integer  NULL DEFAULT NULL  AFTER `tax`;"
+            )
+        );
+    }
+
+
+}
+
+/*
+ * Add transaction_charge column in `erp_acct_pay_bill` table
+ */
+function erp_acct_alter_purchase_details_1_6_9() {
+    global $wpdb;
+
+    $table = $wpdb->prefix . 'erp_acct_purchase_details';
+    $cols  = $wpdb->get_col( "DESC $table" );
+
+    if ( !in_array( 'tax', $cols ) ) {
+        $wpdb->query(
+            $wpdb->prepare(
+                "ALTER TABLE $table ADD `tax` decimal(20,2)  NULL DEFAULT NULL  AFTER `amount`;"
+            )
+        );
+    }
+
+
+}
+
+
+function crate_erp_acct_purchase_details_tax_table_1_6_9() {
+
+    global $wpdb;
+    $sql = "CREATE TABLE IF NOT EXISTS {$wpdb->prefix}erp_acct_purchase_details_tax (
```

Comment:
'IF NOT EXISTS' is not needed for table creation while using dbDelta

Reply:
We decided to follow the previous flow for sql